src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,18 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.8.1
+
+* Fix Submariner healtcheck support with Globalnet.
+* OVN in kind support deployment support `make clusters using=ovn` (E2E testing & development environment).
+* Support debugging in Libreswan cabledriver.
+* Fix the cabledriver label in the latency metrics.
```

Comment:
What was fixed exactly?

Reply:
https://github.com/submariner-io/submariner/pull/1052  We were tagging the metrics with an incorrect label.